application/src/main/resources/db/V12__Add_Bookmarks_System.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE bookmarks
+(
+    author_id            BIGINT    NOT NULL,
+    channel_id           BIGINT    NOT NULL,
+    created_at           TIMESTAMP NOT NULL,
+    note                 TEXT,
+    scheduled_removal_at TIMESTAMP,
```

Comment:
id just go for `remove_at` or `delete_at` or `expires_at`.

Reply:
I will go with delete_at